### ISSUE TITLE
🐛[BUG][System/Equipment] - Return Ring not working

### ISSUE THREAD

**Issue by EsuferLandale** [HorizonFFXI/HorizonXI-Issues#1691]
**What does this bug affect?**

Unable to use Return Ring when in a zone that meets the criteria (zone controlled by home nation, supply quest done, charges on item present)

**Steps to Reproduce**

Using Return Ring while equipped within a home-nation controlled zone
The message "This action cannot be used in this area" will appear in red in the chatlog, a cast bar will start in the top left of the screen that is not cancelled by movement and does not trigger an item usage animation from the character (a charge is not used by this)

**Expected behavior**

Return Ring providing the Outpost Warp effect and warping character to the controlled outpost in the region

**Screenshots**

N/A

**Setup Information**

ver.30221101_1

**Additional context**

A friend who started recently is getting a similar issue, who also started in San d'Oria - I asked on the discord if anyone else was having it and a user was able to confirm that their Return Ring was working. I also relogged and tested it and it still doesn't work for me or my friend.

**Comment by sarca571ca** [comment]
Do you have the OP in the region you're trying to use the ring in? As in can you warp to it using the NPC in Jeuno/Home Nation?

**Comment by EsuferLandale** [comment]
Yep, outpost warp itself is working fine.Sent from my iPhoneOn 28 Jul 2023, at 02:38, Macleod Robert ***@***.***> wrote:﻿
Do you have the OP in the region you're trying to use the ring in? As in can you warp to it using the NPC in Jeuno/Home Nation?

**Comment by Nixxus** [comment]
Confirming this bug when tested with a San d'Oria aligned character in the Ronfaure region while under San d'Oria control
